generators/generator-bot-enterprise-assistant/generators/app/templates/README.md

```diff
@@ -72,17 +156,16 @@ After provisioning your Azure resources and configuring your authentication conn
     - In **Advanced Settings View (json)**, set the app IDs of the skills that will be allowed to communicate with your root bot. For testing you can set this to "*".
```

Comment:
It would be nice if we can also tell them to not use "*" when deploying because that opens the bot up for anyone to call and call it out as a security risk. Developers are a lazy bunch (me included), once something works there is never really a reason to change it until something breaks. :)

Reply:
Removed the * text